src/cmdline_manager.ts

```diff
@@ -43,6 +43,8 @@ export class CommandLineManager implements Disposable {
     public constructor(private main: MainController) {
         eventBus.on("redraw", this.handleRedraw, this, this.disposables);
         this.input = window.createQuickPick();
+        // https://github.com/microsoft/vscode/issues/73904
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         (this.input as any).sortByLabel = false;
```

Comment:
`as any` should be enough to avoid a warning. seems heavy handed to have to add the `// eslint-disable...` whenever we need that escape hatch. `as any` already clearly signals that it's intentional

Reply:
Yeah I agree. I felt icky on this one but this rule offers no customization